src/views/css/packlink-core-override.css

```diff
@@ -0,0 +1,17 @@
+#pl-page .pl-form-group input,
+#pl-page .pl-form-group select {
+  height: 100%;
```

Comment:
height for inputs is fixed in code and should be fixed in all integrations.

Reply:
This was implemented before the recent Core changes and is not needed anymore, as all the inputs have expected heights.